src/Text/Lexer/Core.idr

```diff
@@ -135,11 +135,14 @@ record TokenData a where
   constructor MkToken
   line : Int
   col : Int
+  endLine : Int
+  endCol : Int
   tok : a
 
 export
 Show a => Show (TokenData a) where
   show t = show (line t) ++ ":" ++ show (col t) ++ ":" ++ show (tok t)
+  -- TODO: Fix show
```

Comment:
It's best to remove this or expand on what is missing.

Reply:
Oh thanks, I forgot it was still there for previous experiments. Resolved.